src/v2/Components/NavBar/Menus/DropDownMenu/DropDownMenu.tsx

```diff
@@ -109,6 +132,28 @@ const ViewAllMenuItem = styled(MenuItem).attrs({
 
 ViewAllMenuItem.displayName = "ViewAllMenuItem"
 
-const Links = styled(Box)`
+const Links = styled(Flex).attrs({
+  flexDirection: "column",
+  py: 3,
+  mr: [3, 3, 5, 5],
+})`
   border-right: 1px solid ${color("black10")};
 `
+
+const LettersWrap = styled(Box).attrs({
+  px: 1,
+  height: "85px",
+})``
+
+LettersWrap.displayName = "LettersWrap"
+
+const Letters = styled(ArtistsLetterNav).attrs({
```

Comment:
Whenever we add styled-component styles to an existing component we try to follow the `StyledComponentName` convention; so 
```tsx
const StyledArtistsLetterNav = styled(ArtistsLetterNav)
```